scripts/actions/mobskills/powder_keg.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- Powder Keg (Lion)
+-- Conal damage, knock back, defense down, and magic defense down.
+-- Type: Physical
+-- Skillchain Properties: Fusion/Compression
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 1
+    local accmod = 2
+    local dmgmod = 6
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
+    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.PIERCING, info.hitslanded)
+
+    xi.mobskills.mobPhysicalStatusEffectMove(mob, target, skill, xi.effect.DEFENSE_DOWN, 20, 0, 60)
+    xi.mobskills.mobPhysicalStatusEffectMove(mob, target, skill, xi.effect.DEFENSE_DOWN, 20, 0, 60)
```

Comment:
twice?

Reply:
sorry, one was meant to be magic defense, will update